Libraries/LibIPC/Concepts.h

```diff
@@ -27,54 +27,45 @@ namespace IPC::Concepts {
 
 namespace Detail {
 
+// Cannot use SpecializationOf with these templates because they have non-type parameters. See https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p1985r3.pdf
```

Comment:
I changed the order of declarations, which makes the diff unreadable. [This is a diff that shows changes of substance](https://github.com/LadybirdBrowser/ladybird/commit/aa70a1532824c7) (without reordering or `inline` removing).